### ISSUE TITLE
Why is this package size so big?

### ISSUE THREAD

**Issue by Azd325** [tox-dev/tox-pipenv#38]
Im curious why is this package size so big? :)
```
machtfit-oscar-1s_6MTPE ❯ pip install tox-pipenv
Collecting tox-pipenv
  Downloading tox-pipenv-1.4.0.tar.gz (18.0MB)
    100% |████████████████████████████████| 18.0MB 69kB/s
```

**Comment by tonybaloney** [comment]
I accidentally included a virtualenv in the test/integration/env directory... Will remove for next build

**Comment by Azd325** [comment]
Ahh geat :)

**Comment by tonybaloney** [comment]
Now 15KB. Minor improvement... :-)

**Comment by tonybaloney** [comment]
released in 1.4.1
